### ISSUE TITLE
API access should be part of Matomo analytics

### ISSUE THREAD

**Issue by evyncke** [YangCatalog/backend#536]
All public/authenticated API access should also be counted in the Matomo analytics.

Obviously, the javascript approach cannot be used with an API but (my guess) the backend can issue a web service request to the IETF Matomo server with enough information about the API request (could include at least the API end point).

**Comment by SlavomirMazurPantheon** [comment]
Hello Eric, 
we will try to find out the possibilities of how to track API access or possibly sent requests to IETF Matomo server and we'll get back to you.

**Comment by SlavomirMazurPantheon** [comment]
Hello @evyncke,
individual API applications were updated:
yangre: https://github.com/YangCatalog/yangre/pull/17
yangvalidator: https://github.com/YangCatalog/yang-validator-extractor/pull/101
backend: https://github.com/YangCatalog/backend/pull/545

yangre and yangvalidator had already deployed changes during the last week, and in the report that arrived today it is already possible to see records from the API. backend will start to track API this week too.

Regards,
Slavomír Mazúr

**Comment by evyncke** [comment]
Thank you and indeed the weekly Matomo report included some new URL (but it was unclear to me whether those were linked to the API -- is there any chance to add a title to those API requests)

**Comment by SlavomirMazurPantheon** [comment]
Yes it is possible work with `title` string, I can update it to something like:
`API - /yangvalidator/v2/validate`
or
`API yangvalidator /yangvalidator/v2/validate`
or maybe you have some better option?

**Comment by evyncke** [comment]
I prefer `API yangvalidator /yangvalidator/v2/validate`
